packages/core/src/theme-default/layout/DocLayout/index.module.scss

```diff
@@ -14,6 +14,10 @@
   display: none;
 }
 
+:global(.rspress-doc) {
+  min-height: 600px;
```

Comment:
How about `xx vh`, which is more friendly on small screens.

Reply:
use 60 vh instead